File: UELPMAPI/UELPM.Model/Models/Department.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UELPM.Model.Models
{
    public class DepartmentManagement
    {
        public int LocationId { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int? IsDepartment { get; set; }
        public DateTime CreatedDate { get; set; }
        public int CreatedBy { get; set; }
        public int UpdatedBy { get; set; }
        public DateTime UpdatedDate { get; set; }
        public int CompanyId { get; set; }
        public int? IsDeleted { get; set; }
    }

    public class DepartmentManagementDisplayResult
    {
        public List<DepartmentManagement> DepartmentList { get; set; }
        public int TotalRecords { get; set; }
    }


}

File: UELPMAPP/src/app/administration/models/department.ts
import { GridDisplayInput } from "../../shared/models/shared.model";

export class DepartmentManagement
{
    LocationId : number;
    Name : string;
    Code : string;
    IsDepartment?: number;
    Description : string;
    CreatedDate : Date;
    CreatedBy : number;
    CompanyId : number;
    IsDeleted? : number;
}

export class DepartmentManagementDisplayResult
{
    DepartmentList:Array<DepartmentManagement>;
    TotalRecords :number;
}
